frontend/src/pages/Projects/Project/index.tsx

```diff
@@ -1,14 +1,41 @@
-import React from "react";
+import { TextField } from "@mui/material";
+import React, { useLayoutEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { DetailsList } from "../../../components";
 import { Outlet } from "react-router-dom";
+import { apiAxios } from "../../../utils";
 
 export const Project = () => {
+  const [projectInfo, setProjectInfo] = useState({ id: "dummy" });
+
   const { projectId } = useParams();
 
+  useLayoutEffect(() => {
+    // if ("number" === typeof projectId) {
```

Comment:
Should this commented-out code still be here?

Reply:
yes, it's for the next step in the final code. This is MVP.